docs/docs/migration-guide.mdx

```diff
@@ -126,6 +126,152 @@ confidences. This change can affect the maximum confidence predicted by the
 However, the previous behavior can still be enforced by setting
 `ranking_length=10` and `renormalize_confidences=True`.
 
+### Slot Mappings
+
+As of version 3.0, there is a single explicit mechanism for slot filling enabled by defining slot mappings for each slot
+in the `slots` section of the domain file.
+
+This mechanism replaces the implicit slot setting via auto-fill of slots with entities of the same name.
+The `auto_fill` key in the domain is no longer available, as well as the `auto_fill` parameter in the constructor of
+the `Slot` class.
+
+While forms continue to request the next slot, the extraction of slots is now delegated exclusively to the new default
+action `action_extract_slots`. This action runs in the background after each user turn without it being required to be
+triggered manually.
+
+This approach allows slots to be kept up to date over the course of a conversation, and removes duplicated effort in
```

Comment:
Paragraph starting on line 142 can probably go in the first paragraph (makes it easier to find out what changed and why if you're only skimming/reading part way)